app/javascript/app/components/data-explorer-content/data-explorer-content-selectors.js

```diff
@@ -542,6 +543,15 @@ export const parseData = createSelector([getData], data => {
   return updatedData.map(d => pick(d, whiteList));
 });
 
+export const getFirstColumnHeaders = createSelector([parseData], data => {
+  if (!data || !data.length) return null;
+  const isANumber = i => !isNaN(parseInt(i, 10));
+  const yearColumnKeys = Object.keys(data[0])
+    .filter(k => isANumber(k))
+    .reverse();
+  return DATA_EXPLORER_FIRST_COLUMN_HEADERS.concat(yearColumnKeys);
```

Comment:
Wouldn't it be more accurate to name this as `DATA_EXPLORER_TABLE_HEADERS`? The `first_column` bit could be misleading?